recaf-core/src/main/java/me/coley/recaf/decompile/quilt/QuiltFlowerDecompiler.java

```diff
@@ -35,9 +36,34 @@ public class QuiltFlowerDecompiler extends Decompiler {
 	private final ThreadLocal<String> result = new ThreadLocal<>();
 
 	public QuiltFlowerDecompiler() {
-		super("QuiltFlower", "1.9.0");
+		super("VineFlower", findQuiltFlowerVersion());
 		fernFlowerProperties.put("ind", "    ");
 	}
+    private static String findQuiltFlowerVersion() {
+        try {
+            ClassLoader classLoader = Fernflower.class.getClassLoader();
+            if (classLoader != null) {
+                Enumeration<URL> resources = classLoader.getResources("META-INF/MANIFEST.MF");
+				while (resources.hasMoreElements()) {
```

Comment:
This won't work in JAR release form. The reason why this works in the IDE is because each dependency is loaded from its own JAR file. If we combine the jars, and replace things like the `MANIFEST.MF` with our own then the information this relies on will no longer exist.